src/ruby/ext/grpc/extconf.rb

```diff
@@ -101,7 +126,9 @@
 end
 
 $LDFLAGS << ' -Wl,-wrap,memcpy' if linux
-$LDFLAGS << ' -static-libgcc -static-libstdc++' if linux
+if linux && RUBY_ENGINE != 'truffleruby'
```

Comment:
nit: can we please add a comment here why this can't work on truffleruby?

Reply:
Done, and I checked it works with the current code and doesn't without (`External LLVMFunction _ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info cannot be found. (RuntimeError)`)